.github/workflows/pr.yaml

```diff
@@ -13,11 +13,11 @@ jobs:
     needs:
       - python-gpu-tests
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.06
+    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.08
 
   python-gpu-tests:
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-24.06
+    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-24.08
```

Comment:
Agreed, lets also change the base image to `rapidsai/base:24.08-cuda12.2-py3.11`